importer/views.py

```
@@ -27,25 +31,45 @@
-    rootdir = f"{str(Path.home())}/input"
+
+def get_input_root_dir():
+    """
+    Determine the input root directory.
+
+    Returns:
+        Path to the input directory (/input if running in Docker, otherwise ~/input).
+    """
+    if Path("/input").is_dir():
+        return "/input"
+    return f"{str(Path.home())}/input"
```

Ruff (0.15.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)